app/models/group_offer.rb

```diff
@@ -17,8 +18,12 @@ class GroupOffer < ApplicationRecord
 
   has_many :feedbacks, as: :feedbackable, dependent: :destroy
 
+  delegate :department_manager?, to: :user
+
   validates :title, presence: true
   validates :necessary_volunteers, numericality: { greater_than: 0 }, allow_nil: true
+  validate :department_manager_has_department?, if: :department_manager?
+  validates :department, presence: true, if: :department_manager?
```

Comment:
Since the department is required we should add this to the `:department_manager` factory, and also remove the `:with_departments` trait if other users can't have departments (not sure)

Reply:
has been discussed